.github/workflows/canister-tests.yml

```diff
@@ -184,22 +187,25 @@ jobs:
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('src/**/Cargo.lock', 'rust-toolchain.toml') }}-2
 
-      - name: Create fake assets
```

Comment:
Just like the CAPTCHA related code in #832 , I suspect (though can't quite remember) that we used to build the II code in this workflow; this is not the case now anyway and this step is not necessary anymore.